src/scss/includes/masqAvatar.scss

```diff
@@ -15,5 +15,6 @@
 
   &-image {
     width: 100%;
+    height: 100%;
```

Comment:
`object-fit` could also be used to preserve the aspect ratio:

```suggestion
    height: 100%;
    object-fit: cover;
```

Reply:
:+1: